dockerfile_scripts/install_deb_packages.sh

```diff
@@ -0,0 +1,37 @@
+apt-get update \
+	&& DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+		autoconf \
+		automake \
+		autotools-dev \
+		build-essential \
+		ca-certificates \
+		curl \
+		daemontools \
+		debhelper \
+		devscripts \
+		ibverbs-providers \
+		libibverbs1 \
```

Comment:
these 4 are different from the Dockerfile-base-cpu, and they seem to come from the Dockerfile-base-gpu.  Is that accurate?  Why not update Dockefile-base-gpu to use install_deb_packages as well?  There is still a scary amount of code duplication in this repo, and if you're going to refactor new additions, it'd be real nice if you could backport those fixes to the older dockerfiles as well